templates/task-sc.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+    name: skopeo-copy
```

Comment:
Could you please check the settings of your editor? It should have the tab-stop set to two spaces.

Reply:
Okay will edit it.